feat(preview): Add `Preview` Component (#108)
zenorocha/react-email: packages/preview/readme.md

> ![React Email img cover](https://react-email-assets.vercel.app/preview.png)

Reviewer: ```suggestion
![React Email preview cover](https://react-email-assets.vercel.app/preview.png)
```

<div align="center"><strong>@react-email/preview</strong></div>

> <div align="center">Text that appears under the subject line in the inbox.</div>

Reviewer: ```suggestion
<div align="center">A preview text that will be displayed in the inbox of the recipient.</div>
```

Author: I like it!
